### ISSUE TITLE
Categorizing Links into Different Categories

### ISSUE THREAD

**Issue by ConsenSys-Academy** [ConsenSys-Academy/Blockchain-Developer-Bootcamp#107]
Creating more descriptive groups for links provided in sections

**Comment by dlsso** [comment]
Note: specifically intended to categorize links to help students decide **where to invest their time**, since it appears doing all "additional materials" links would be well over 40 hours per week.

In my mind I've been dividing them into
- Rehash of core material
- Expands on core material
- Bonus (non-essential or tangentially related, and  like anonymity)

If instructors could categorize by **priority** that would be another extremely helpful way to organize them though. (Maybe something like: highly recommended, recommended if you have time, just for fun/enrichment)

**Comment by ConsenSys-Academy** [comment]
@dlsso These are great points. Any reason why we wouldn't use your first series of categories?

> In my mind I've been dividing them into
> 
> Rehash of core material
> Expands on core material
> Bonus (non-essential or tangentially related, and like anonymity)

**Comment by dlsso** [comment]
Those tell you **what** pretty well, but for the purposes of deciding where to invest time, knowing what the course authors think is important would be very helpful as well.

Getting any categorization in is much more important than making it perfect though. In chapter 1 alone there are well over 100 external links, and total time on those looks like it could be a couple hundred hours. Anything that helps filter them would be extremely valuable.

**Comment by ConsenSys-Academy** [comment]
@dlsso Anything you might be able to help with? We won't be able to start this in earnest until end of the cohort but seems like you may have already done some of this work, even if you haven't written it down

**Comment by dlsso** [comment]
Yeah I haven't tracked them. Doing so in a helpful way would be a little tricky. Maybe a PR with code comments?
